apps/desktop-wallet/src/features/modals/modalTypes.ts

```diff
@@ -16,7 +16,7 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
```

Comment:
- [x] Unknown tokens inconsistent display:
   - [x] both + and - displayed at the same time
   - [x] label has diff styles than the rest of the modal rows
   - [x] displayed on the left and not on the right

<img width="300" alt="image" src="https://github.com/user-attachments/assets/5f4bb6fa-907e-4d40-82b6-911643c99891" />

Reply:
Addressed in my PR.